elixir/apps/web/lib/web/auth.ex

```diff
@@ -169,10 +207,14 @@ defmodule Web.Auth do
     {:cont, mount_subject(socket, params, session)}
   end
 
+  def on_mount(:mount_account, params, session, socket) do
+    {:cont, mount_account(socket, params, session)}
+  end
+
   def on_mount(:ensure_authenticated, params, session, socket) do
     socket = mount_subject(socket, params, session)
 
-    if socket.assigns.subject do
+    if socket.assigns[:subject] do
```

Comment:
If the subject is not assigned before the function will just return whatever is in `else` block, but now it will raise. I don't think it should be ever called when the subject is not assigned, so it will help to fight some accidental mistakes.

Reply:
Ah, ok, interesting.